src/Command/ErrorFormatter/TableErrorFormatter.php

```diff
@@ -13,14 +13,17 @@ class TableErrorFormatter implements ErrorFormatter
 	private RelativePathHelper $relativePathHelper;
 
 	private bool $showTipsOfTheDay;
+	private ?string $editorUrl;
```

Comment:
```suggestion
	private bool $showTipsOfTheDay;

	private ?string $editorUrl;
```

Will fix https://github.com/phpstan/phpstan-src/pull/515/checks?check_run_id=2594792188#step:7:54